structures/occupancytypes.go

```diff
@@ -301,6 +302,34 @@ func res11swb() OccupancyTypeStochastic {
 	sdf.DamageFunctions[hazards.Depth] = structuredamagefunctionStochastic
 	cdf.DamageFunctions[hazards.Depth] = contentdamagefunctionStochastic
 
+	salinityxs := []float64{-4.0, -3.0, -2.0, -1.0, 0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0, 11.0, 12.0, 13.0, 14.0, 15.0, 16.0}
+	salinityydists := make([]statistics.ContinuousDistribution, 21)
+	salinityydists[0], _ = statistics.InitDeterministic(0.0)
+	salinityydists[1], _ = statistics.InitDeterministic(0.0)
+	salinityydists[2], _ = statistics.Init([]float64{2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14}, []int64{903, 3461, 3229, 5151, 4727, 4549, 4207, 3841, 3594, 3278, 2979, 2666, 2387})
+	salinityydists[3], _ = statistics.Init([]float64{3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16}, []int64{85, 2106, 1985, 1765, 2983, 2852, 2680, 2501, 2299, 2160, 1957, 1829, 1684, 1502})

```

Comment:
It might be dangerous to ignore the errors. I know you created the data and sanitized it, so it should be ok to underbar the error and ignore them. It does generally violate idiomatic go though.